lib/wanda_web/controllers/execution_controller.ex

```diff
@@ -47,6 +54,15 @@ defmodule WandaWeb.ExecutionController do
       404 => OpenApiSpex.JsonErrorResponse.response()
     }
 
+  operation :start,
+    summary: "Start a Checks Execution",
+    description: "Trigger a Checks Execution on the target infrastructure",
```

Comment:
To match the description better -> `Start a Checks Execution on the target infrastructure`